### ISSUE TITLE
ping -4 parsing returns invalid destination_ip

### ISSUE THREAD

**Issue by pschmitt** [kellyjonbrazil/jc#407]
When using the `-4` flag for ping the `destination_ip` that jc returns is wrong.

```shell
» ping -c 1 geclat-ts.heimat.dev | jc --ping
{"destination_ip":"fd7a:115c:a1e0:ab12:4843:cd96:6244:697c","data_bytes":56,"pattern":null,"destination":"geclat-ts.heimat.dev","packets_transmitted":1,"packets_received":1,"packet_loss_percent":0.0,"duplicates":0,"time_ms":0.0,"round_trip_ms_min":0.028,"round_trip_ms_avg":0.028,"round_trip_ms_max":0.028,"round_trip_ms_stddev":0.0,"responses":[{"type":"reply","timestamp":null,"bytes":64,"response_ip":"fd7a:115c:a1e0:ab12:4843:cd96:6244:697c","icmp_seq":1,"ttl":64,"time_ms":0.028,"duplicate":false}]}
» ping -6 -c 1 geclat-ts.heimat.dev | jc --ping
{"destination_ip":"fd7a:115c:a1e0:ab12:4843:cd96:6244:697c","data_bytes":56,"pattern":null,"destination":"geclat-ts.heimat.dev","packets_transmitted":1,"packets_received":1,"packet_loss_percent":0.0,"duplicates":0,"time_ms":0.0,"round_trip_ms_min":0.046,"round_trip_ms_avg":0.046,"round_trip_ms_max":0.046,"round_trip_ms_stddev":0.0,"responses":[{"type":"reply","timestamp":null,"bytes":64,"response_ip":"fd7a:115c:a1e0:ab12:4843:cd96:6244:697c","icmp_seq":1,"ttl":64,"time_ms":0.046,"duplicate":false}]}
» ping -4 -c 1 geclat-ts.heimat.dev | jc --ping
{"destination_ip":"56","data_bytes":84,"pattern":null,"destination":"ping","packets_transmitted":1,"packets_received":1,"packet_loss_percent":0.0,"duplicates":0,"time_ms":0.0,"round_trip_ms_min":0.021,"round_trip_ms_avg":0.021,"round_trip_ms_max":0.021,"round_trip_ms_stddev":0.0,"responses":[{"type":"reply","timestamp":null,"bytes":64,"response_ip":"100.68.105.124","icmp_seq":1,"ttl":64,"time_ms":0.021,"duplicate":false}]}
```

Here's the raw output:

```shell
» ping -4 -c 1 geclat-ts.heimat.dev
PING  (100.68.105.124) 56(84) bytes of data.
64 bytes from 100.68.105.124 (100.68.105.124): icmp_seq=1 ttl=64 time=0.040 ms

---  ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.040/0.040/0.040/0.000 ms
» ping -6 -c 1 geclat-ts.heimat.dev
PING geclat-ts.heimat.dev(geclat.snake-eagle.ts.net (fd7a:115c:a1e0:ab12:4843:cd96:6244:697c)) 56 data bytes
64 bytes from geclat.snake-eagle.ts.net (fd7a:115c:a1e0:ab12:4843:cd96:6244:697c): icmp_seq=1 ttl=64 time=0.041 ms

--- geclat-ts.heimat.dev ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.041/0.041/0.041/0.000 ms
```

**Comment by kellyjonbrazil** [comment]
Thanks for reporting this - I'll take a look!

**Comment by kellyjonbrazil** [comment]
Hi there - I was testing this on a couple of linux VMs and couldn't reproduce the `ping` output you were getting with the `-4` option. Curious what OS and `ping` version you have? I was testing on Centos 7 with iputils-s20160308.

**Comment by pschmitt** [comment]
That's Arch Linux with iproute2 6.2.0-2, but that does not really matter here I believe. 

Tailscale Magic DNS is probably what triggers this behavior.

**Comment by kellyjonbrazil** [comment]
Ah ok - then I'll just use your output sample to make the fix. Thanks!

**Comment by kellyjonbrazil** [comment]
I have a fix in the `dev` branch ready for testing. You can copy the following file to your plugin folder:
https://github.com/kellyjonbrazil/jc/blob/dev/jc/parsers/ping.py

- Linux/unix: `$HOME/.local/share/jc/jcparsers`
- macOS: `$HOME/Library/Application Support/jc/jcparsers`
- Windows: `$LOCALAPPDATA\jc\jc\jcparsers`

Let me know if you run into any further issues!

**Comment by pschmitt** [comment]
Thanks for following up on this! <3

LGTM:

```
» pipx uninstall jc; pipx install git+https://github.com/kellyjonbrazil/jc@dev
» for i in -4 -6; do ping $i -c 1 geclat-ts.heimat.dev | tee $(tty) | jc --ping --pretty; done
PING  (100.68.105.124) 56(84) bytes of data.
64 bytes from geclat.schmitt.co.beta.tailscale.net (100.68.105.124): icmp_seq=1 ttl=64 time=0.016 ms

---  ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.016/0.016/0.016/0.000 ms
{
  "destination_ip": "100.68.105.124",
  "data_bytes": 56,
  "pattern": null,
  "packets_transmitted": 1,
  "packets_received": 1,
  "packet_loss_percent": 0.0,
  "duplicates": 0,
  "time_ms": 0.0,
  "round_trip_ms_min": 0.016,
  "round_trip_ms_avg": 0.016,
  "round_trip_ms_max": 0.016,
  "round_trip_ms_stddev": 0.0,
  "responses": [
    {
      "type": "reply",
      "timestamp": null,
      "bytes": 64,
      "response_ip": "100.68.105.124",
      "icmp_seq": 1,
      "ttl": 64,
      "time_ms": 0.016,
      "duplicate": false
    }
  ]
}
PING geclat-ts.heimat.dev(geclat.schmitt.co.beta.tailscale.net (fd7a:115c:a1e0:ab12:4843:cd96:6244:697c)) 56 data bytes
64 bytes from geclat.schmitt.co.beta.tailscale.net (fd7a:115c:a1e0:ab12:4843:cd96:6244:697c): icmp_seq=1 ttl=64 time=0.025 ms

--- geclat-ts.heimat.dev ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.025/0.025/0.025/0.000 ms
{
  "destination_ip": "fd7a:115c:a1e0:ab12:4843:cd96:6244:697c",
  "data_bytes": 56,
  "pattern": null,
  "destination": "geclat-ts.heimat.dev",
  "packets_transmitted": 1,
  "packets_received": 1,
  "packet_loss_percent": 0.0,
  "duplicates": 0,
  "time_ms": 0.0,
  "round_trip_ms_min": 0.025,
  "round_trip_ms_avg": 0.025,
  "round_trip_ms_max": 0.025,
  "round_trip_ms_stddev": 0.0,
  "responses": [
    {
      "type": "reply",
      "timestamp": null,
      "bytes": 64,
      "response_ip": "fd7a:115c:a1e0:ab12:4843:cd96:6244:697c",
      "icmp_seq": 1,
      "ttl": 64,
      "time_ms": 0.025,
      "duplicate": false
    }
  ]
}
```

**Comment by kellyjonbrazil** [comment]
Released in v1.23.3
